R/helpers.R

```diff
@@ -402,7 +405,8 @@ get_hover <- function(hn, data) {
   for(ii in seq_len(ncol(data)))
     data[[ii]] <- format(data[[ii]])
 
-  hdict <- lapply(seq_along(hn), function(ii) list(hn[ii], paste("@", hn2[ii], sep = "")))
+  hdict <- lapply(seq_along(hn), function(ii) list(gsub("hover_", "", hn[ii]), 
```

Comment:
Correct me if I'm wrong, but once the data dict was created, it didn't seem that the prefixed labels were necessary.  I'm finding that the prefix makes the hover text wider, which can cause readability problems with grid plots.  If the prefix is necessary, I'd suggest something more concise if possible.